lib/services.js

```diff
@@ -450,6 +450,7 @@ export default {
             metadata.putObjectMD(bucketName, longMPUIdentifier,
                 multipartObjectMD, log, (err) => {
                     if (err) {
+                        log.error('error from metadata', { error: err });
```

Comment:
That's what I mean, I want to make your `perhaps` into a certainty :)
Also, this will help you learn S3's code by digging into it anyways